src/index.js

```diff
@@ -9,7 +9,11 @@ import * as base from '@most/prelude'
 import { of, empty, never } from './source/core'
 import { from } from './source/from'
 import { periodic } from './source/periodic'
-import symbolObservable from 'symbol-observable'
+import provideSymbolObservable from 'symbol-observable/ponyfill'
+
+// eslint-disable-next-line no-new-func
+const globalThis = new Function('return this')()
```

Comment:
Probably should revisit this using the `globalthis` shim so this will continue to work under a no-eval CSP.